build.sh

```diff
@@ -61,7 +61,8 @@ stop_postgresql # in case it's running from a previous build
 start_postgresql
 
 # Run the tests.
-bazel test //... \
+bazel test //ledger/participant-integration-api:participant-integration-api-tests_test_suite_src_test_suite_scala_platform_store_dao_JdbcLedgerDaoPostgresqlSpec.scala \
+  --test_arg=-z --test_arg="fall back to limit-based query with consistent results" \
```

Comment:
You need to make this change in _build.ps1_ too.

Reply:
why? we’re not going to merge this, we just want to see the error.

Reply:
So you can see the error on Windows without other random timeouts. 🙃

Reply:
We saw this error on macos and Linux, I would go as far as just disabling the Windows job for testing here.